packages/@aws-cdk-testing/framework-integ/test/pipelines/test/integ.newpipeline-with-allPrepareNodesFirst.ts

```diff
@@ -0,0 +1,81 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+/// !cdk-integ PipelineStack pragma:set-context:@aws-cdk/core:newStyleStackSynthesis=true
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { App, Stack, StackProps, Stage, StageProps } from 'aws-cdk-lib';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
+import * as pipelines from 'aws-cdk-lib/pipelines';
+import { Construct } from 'constructs';
+
+class PipelineStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const pipeline = new pipelines.CodePipeline(this, 'Pipeline', {
+      synth: new pipelines.ShellStep('Synth', {
+        input: pipelines.CodePipelineSource.gitHub('Nico-DB/aws-cdk', 'main'),
+        commands: ['npm ci', 'npm run build', 'npx cdk synth'],
+      }),
+      allPrepareNodesFirst: true,
+    });
+
+
```

Comment:
Build is yelling at you here for having multiple blank lines.

Reply:
has been resolved